File: C/Estrutura Sequencial/Troco.c
#include <stdio.h>
#include <string.h>



int main ()
{

    double preco, valor, troco;
    int qtde;

    printf ("Preço unitario do produto: ");
    scanf ("%lf", &preco);
    printf ("Quantidade comprada:");
    scanf ("%d", &qtde);
    printf ("Dinheiro recebido: ");
    scanf ("%lf", &valor);

    troco = valor - (preco * qtde);

    printf ("TROCO = %.2lf\n", troco);

    return 0;
}

File: Python/Estrutura Sequencial/Troco.py
preco: float; dinheiro: float; troco: float
qtde: int

preco = float(input("Preço unitário do produto: "))
qtde = int(input("Quantidade comprada: "))
dinheiro = float(input("Dinheiro recebido: "))


troco = dinheiro - (preco * qtde)


print(f"TROCO: {troco:.2f}")